HttpConnection.php

```diff
@@ -689,7 +692,7 @@ function putRequest($url, $type = 'none', $file = NULL) {
     // Ugly substitute for a try catch finally block.
     $exception = NULL;
     try {
-      $results = $this->doCurlRequest();
+      $results = $this->doCurlRequest(NULL, $fh);
```

Comment:
@jonathangreen @adam-vessey the absence of $fh seems to be handled by the argument 's [default]( https://github.com/Islandora/tuque/pull/153/commits/213a90bcffa7fc687f8013171110cdc6a2124ecb#diff-2042586689e534b400950d72a360fa29R446)

This seems to consistent with the pre-pull way of doing it for `$file`, but I feel that maybe being explicit on a NULL `$fh` on type `none` would not harm at all.